tests/test_query_engine.py

```diff
@@ -19,9 +20,10 @@ def test_year_from_date(engine):
     )
 
     patients = SelectPatientTable("patients")
+    registrations = SelectTable("practice_registrations")
 
     class DatasetDefinition:
-        population = AggregateByPatient.Exists(SelectColumn(patients, "patient_id"))
+        population = AggregateByPatient.Exists(registrations)
```

Comment:
Do we not have a test for `Count`?

Reply:
We do, but it's in `legacy/test_query_engine.py` and so it's handled by the translation code.

The only reason this test needs updating is that it happens to use `Exists` for the population definition (and that's because we need a definition that's supported by the existing Query Engine).